api4/user.go

```diff
@@ -1904,7 +2021,9 @@ func enableUserAccessToken(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.SetInvalidParam("token_id")
 	}
 
-	c.LogAudit("")
+	auditRec := c.MakeAuditRecord("enabledUserAccessToken", audit.Fail)
```

Comment:
```suggestion
	auditRec := c.MakeAuditRecord("enableUserAccessToken", audit.Fail)
```

Reply:
Should we create a go vet check for verify the execution of success at least once, and the values for `MakeAuditRecord`? I mean, the values must be always the name of the container function and `audit.Fail`, right?

Reply:
I had the same thought, and filed a ticket in JIRA https://mattermost.atlassian.net/browse/MM-23113